nfts/frontend/src/App.js

```diff
@@ -14,6 +15,7 @@ function App() {
         <Navbar />
         <Routes>
           <Route path="/" element={<TransferNFT sdk={sdk} />} />
+          <Route path="/nftList" element={<ListNFT sdk={sdk} />} />
```

Comment:
by default let's open the List nft page

Reply:
Done

Reply:
Let's delete the transfer page. we don;t need it